src/components/screen.cpp

```diff
@@ -107,7 +108,12 @@ void screen::handle(const evt::randr_screen_change_notify& evt) {
   if (screen->width_in_pixels != m_size.w || screen->height_in_pixels != m_size.h) {
     changed = true;
   } else {
-    changed = have_monitors_changed();
+    auto p = have_monitors_changed();
```

Comment:
We should use structured binding here for clarity. Something like this:

```
    auto [changed, num_monitors] = have_monitors_changed();
```

Not sure how that behaves though since `changed` is already defined, so maybe something like this:

```
    int num_monitors = 0;
    std::tie(changed, num_monitors) = have_monitors_changed();
```

Or this:

```
    auto [changed_, num_monitors] = have_monitors_changed();
    changed = changed_
```

Reply:
I did not know about destructed binding. TIL :)